data/schema.sql

```diff
@@ -1,26 +1,25 @@
-DROP DATABASE IF EXISTS agents;
-CREATE DATABASE IF NOT EXISTS agents;
+DROP DATABASE agents;
```

Comment:
You should include back the DROP DATABASE IF EXISTS agents; because when i used that the schema.sql was loading into the dab